src/main/resources/api.yaml

```diff
@@ -207,3 +270,15 @@ definitions:
     required:
       - enrollmentPeriod
       - durationPeriod
+
+  UserAvailabilityDto:
+    type: object
+    properties:
+      sprintDurationPeriod:
+        $ref: '#/definitions/LocalDateTimePeriodDto'
+      availableMinutes:
```

Comment:
Ja osobiście bym dał `timeAvailable`, `timeRemaining` i w description napisał, że to minuty.

Reply:
okej, robi sens